docs-content/installing.html.md.erb

```diff
@@ -31,64 +31,93 @@ This adds the tile to your staging area.
 
-![Conjur Service Broker Configuration tab](service_broker_config_screen.png)
+![Conjur Service Broker Configuration tab](tile_config_screen.png)
 
-  * **Conjur Account**: The organization account assigned during Conjur appliance installation. If you are using a hosted Conjur instance for a proof of concept, the account is typically your email address.
+  * **Conjur Account**:
+    The organization account assigned during Conjur appliance installation. If
+    you are using the [**Try Conjur**](https://www.conjur.org/get-started/try-conjur.html) hosted Conjur instance for a proof of concept, the account is
+    typically your email address.
 
-  * **Conjur Appliance URL**: The URL of the Conjur appliance that you are connecting to.
+  * **Conjur Appliance URL**:
+    The URL of the Conjur appliance that you are connecting to. If you
+    are using [**Try Conjur**](https://www.conjur.org/get-started/try-conjur.html) hosted Conjur,
+    this will be `https://eval.conjur.org`.
 
-  * **Conjur Login**: The fully-qualified ID of a Conjur Host.
+  * **Conjur Follower URL (Optional)**:
+    If using high availability in Conjur EE, this should be the URL
+    for a load balancer that manages the cluster's Follower instances. This is the URL that
+    applications that bind to the service broker will use to communicate with Conjur.
 
+  * **Conjur Login**:
+    The fully-qualified ID of a Conjur host for the service broker.
+    <br/><br/>
     The Host must have `create` and `update` privileges on the dedicated Conjur policy for PCF that you enter in the **PCF Conjur Policy Branch ID** field. If you are not using a dedicated Conjur policy for PCF, this login Host must have `create` and `update` privileges on the Conjur root policy.
-
+    <br/><br/>
     This login is only used by the Service Broker to add and remove Hosts from Conjur policy as your applications are deployed to or removed from PCF.
+    <br/><br/>
+    This entry should be of the form `host/host-id`, where `host-id` is the fully-qualified Conjur Host ID (e.g. `host/pcf-service-broker`). The `host/` prefix indicates to the Conjur authenticator that these credentials belong to a Host, and not a User.
 
-    This entry should be of the form `host/host-id`, where `host-id` is the fully-qualified Conjur Host ID. The `host/` prefix indicates to the Conjur authenticator that these credentials belong to a Host, and not a User.
-
-  * **Conjur API Key**: The API key of the Conjur Host whose identity you provided in the **Conjur Login** field.
-
+  * **Conjur API Key**:
+    The API key of the Conjur Host whose identity you provided in the **Conjur Login** field.
+    <br/><br/>
     If the Host's API key changes, you need to update this field and click **Apply Changes** to continue using the Service Broker to bind applications. After binding, an application has its own credentials for connecting to the Conjur appliance.
```

Comment:
If the Host's API key changes, update this field and click **Apply Changes** to continue using the Service Broker to bind applications.